DatadogCrashReporting/Sources/CrashContext/CrashContextProvider.swift

```diff
@@ -155,6 +165,17 @@ extension CrashContextCoreProvider: FeatureMessageReceiver {
             }
         }
     }
+
+    private func updateRUMAttributes(with baggage: FeatureBaggage, to core: DatadogCoreProtocol) {
+        queue.async { [weak core] in
+            do {
+                self.rumAttributes = try baggage.decode(type: GlobalRUMAttributes.self)
```

Comment:
shouldn't we weakify self as well? 🤔

Reply:
👍 Good point, ✅ updated in the whole file